install_devconsole/create_user.sh

```diff
@@ -35,6 +35,7 @@ EOF
 
 sleep 10s
 oc create clusterrolebinding ${USERNAME}_role --clusterrole=self-provisioner --user=${USERNAME}
+oc adm policy add-role-to-user view ${USERNAME}
```

Comment:
Is this `view` only on the `openshift` namespace?

Reply:
It is in all namespaces